scripts/setup/terminate-psql-sessions

```diff
@@ -26,11 +26,16 @@ root)
     ;;
 esac
 
+if [[ -n "${PGDATABASE}" ]] && [[ -n "${PGHOST}" ]] && [[ -n "${PGUSER}" ]]; then
+    psql=(psql)
+fi
+
 "${psql[@]}" -v ON_ERROR_STOP=1 <<EOF
 SELECT pg_terminate_backend(s.pid)
     FROM pg_stat_activity s, pg_roles r
     WHERE
         s.datname IN ($tables)
         AND r.rolname = CURRENT_USER
-        AND (s.usename = r.rolname OR r.rolsuper = 't');
+        AND (s.usename = r.rolname OR r.rolsuper = 't')
+        AND s.pid <> pg_backend_pid();
```

Comment:
If I'm reading it correctly, it's just meant to avoid trying to terminate ourself, right?  If so, it should be its own commit explaining what it does.

Reply:
Done.